content/en/synthetics/guide/popup.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Handling popups in browser tests
+kind: guide
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/browser-tests/'
+      tag: 'Blog'
+      text: 'User experience monitoring with Datadog Browser Tests'
+    - link: 'synthetics/browser_tests'
+      tag: 'Documentation'
+      text: 'Configure a Browser Test'
+---
+## JavaScript and Basic Auth modals
+
+Synthetic browser tests automatically handle [JavaScript modals][1]: 
+ - `alert` modals are closed. 
+ - `prompt` and `confirm` modals are answered `Lorem Ipsum`. 
+For Basic Auth popups, specify the associated credentials in your browser test configuration, below the [`HTTP Auth` **Advanced option**][2]:
+
+{{< img src="synthetics/guide/popup/http_auth_option.png" alt="Basic Auth Popup">}}
+
+## Application popups
+
+### Anchored popups
+If a popup appears at a specific point of your journey, you can record a step to close it and allow this step to fail using the [corresponding option][3]. This way, your test knows how to behave in case a popup appears. If the popup does not show up, the step fails without causing the whole test to fail. 
```

Comment:
```suggestion

If a popup appears at a specific point of your journey, you can record a step to close it and allow this step to fail using the [corresponding option][3]. This way, your test knows how to behave in case a popup appears. If the popup does not show up, the step fails without causing the whole test to fail. 
```